### DISCUSSION TITLE
Parallax Occlusion Mapping and PBR

### DISCUSSION THREAD

**Discussion by TheArtifct** [IrisShaders/Iris#526] [Q&A] [answered]
I really appreciate the work you do and everything is working very nicely thus far.
I have one question though, I can't seem to get POM or PBR to work. Is it not supported yet or am I doing something wrong? 
I tried by editing the shader files manually and I tried by using the starline fork to access the options but neither method worked.

**Comment by Heath123** [comment, marked as answer]
https://github.com/IrisShaders/Iris/issues/130
